File: testprojects/scala_main/pipelines/SCALA_UNITY_CATALOG/code/src/main/scala/io/prophecy/pipelines/scala_unity_catalog/graph/src_parquet_unity_catalog.scala
package io.prophecy.pipelines.scala_unity_catalog.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_unity_catalog.config.ConfigStore._
import io.prophecy.pipelines.scala_unity_catalog.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object src_parquet_unity_catalog {

  def apply(context: Context): DataFrame =
    context.spark.read
      .format("parquet")
      .load("dbfs:/Prophecy/qa_data/parquet/ThreePartitionsDataset.parquet")

}

File: testprojects/master_python_mainone/pipelines/PYTHON_UC/code/python_uc/graph/src_parquet_unity_catalog.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from python_uc.config.ConfigStore import *
from python_uc.udfs.UDFs import *

def src_parquet_unity_catalog(spark: SparkSession) -> DataFrame:
    return spark.read.format("parquet").load("dbfs:/Prophecy/qa_data/parquet/ThreePartitionsDataset.parquet")
